gitlabform/gitlab/pipelines.py

```diff
@@ -0,0 +1,14 @@
+from gitlabform.gitlab.core import GitLabCore
+
+
+class GitLabPipelines(GitLabCore):
+
+    def get_pipelines(self, project_and_group_name, branch):
+        pipelines = self._make_requests_to_api("projects/%s/pipelines?ref=%s", (project_and_group_name, branch))
+        return pipelines
+
+    def retry_pipeline(self, project_and_group_name, pipeline_id):
+        pipeline = self._make_requests_to_api("projects/%s/pipelines/%s/retry",
```

Comment:
Please return `pipeline` from this method. This will allow you to check if this pipeline succeeded by only querying this single pipeline by id.

Of course to be able to do that you should also add `get_pipeline` (single one, by id) method to this class.

Reply:
Added as requested.
BTW in our case we are taking last pipeline, so for get_pipelines default sorting is descending.